spec/models/abuse_report_spec.rb

```diff
@@ -340,6 +340,12 @@
       expect(spam_report.errors[:base]).to include("This report looks like spam to our system!")
     end
 
+    it "is valid even if the email casing is different" do
+      legit_user.email = legit_user.email.upcase
+      User.current_user = legit_user
+      expect(safe_report.save).to be_truthy
+    end
+
```

Comment:
This looks like it ended up on the wrong branch, did you mean to to add this to https://github.com/otwcode/otwarchive/pull/4755? (If yes, it worries me a bit that the test didn't fail on this branch. It suggests that it's not testing what it's meant to.)